controllers/activemqartemis_reconciler.go

```diff
@@ -2574,20 +2559,24 @@ func getValidCondition(cr *brokerv1beta1.ActiveMQArtemis) metav1.Condition {
 }
 
 func getDeploymentCondition(cr *brokerv1beta1.ActiveMQArtemis, podStatus olm.DeploymentStatus) metav1.Condition {
-	if cr.Spec.DeploymentPlan.Size == 0 {
+	deploymentSize := DefaultDeploymentSize
+	if cr.Spec.DeploymentPlan.Size != nil {
+		deploymentSize = *cr.Spec.DeploymentPlan.Size
+	}
```

Comment:
```suggestion
	deploymentSize := getDeploymentSize(cr)
```
Could `getDeploymentSize` be used here?

Reply:
Indeed, this is a refactor leftover. Thanks for the catch.